microshift_running_apps/microshift-embedded-apps-on-rhel-edge.adoc

```diff
@@ -0,0 +1,56 @@
+:_content-type: ASSEMBLY
+[id="microshift-embedded-apps-on-rhel-edge"]
+= Options for embedding {product-title} applications in a {op-system-ostree} image
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-embedded-apps-on-rhel-edge
+
+toc::[]
+
+You can embed microservices-based workloads and applications in a {op-system-ostree-first} image to run in a {product-title} cluster. Embedded applications can be installed directly on edge devices to run in air-gapped, disconnected, offline environments.
+
+[id="microshift-add-app-RPMs-to-rpm-ostree-image_{context}"]
+== Add application RPMs to an rpm-ostree image
+If you have an application that includes APIs, container images, and configuration files for deployment such as manifests, you can build application RPMs. You can then add the RPMs to your {op-system-ostree} system image.
+
+The following is an outline of the procedures to embed applications or workloads in an fully self-contained operating system image:
+
+* Build your own RPM that includes your application manifest.
+* Add the RPM to the blueprint you used to install {product-title}.
+* Add the workload container images to the same blueprint.
+* Create a bootable ISO.
+
+For a step-by-step tutorial about preparing and embedding applications in a {op-system-ostree} image, see:
+
+* xref:../microshift_running_apps/microshift-embedding-apps-tutorial.adoc#microshift-embedding-apps-tutorial[Embedding applications tutorial]
+
+[id="microshift-add-app-manifests-to-image_{context}"]
+== Add application manifests to an image for offline use
+If you have a simple application that includes a few files for deployment such as manifests, you can add those manifests directly to a {op-system-ostree} system image.
+
+See the "Create a custom file blueprint customization" section of the following {op-system-ostree} documentation for an example:
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/composing_installing_and_managing_rhel_for_edge_images/composing-a-rhel-for-edge-image-using-image-builder-command-line_composing-installing-managing-rhel-for-edge-images#image-customizations_composing-a-rhel-for-edge-image-using-image-builder-command-line[Create a custom file blueprint customization]
+
+[id="microshift-embed-apps-for-offline-use_{context}"]
+== Embed applications for offline use
+If you have an application that includes more than a few files, you can embed the application for offline use. See the following procedure:
```

Comment:
I haven't done this because of the guidance not to use lead-in sentences for xrefs or links. Can't find the link now, of course. But I have gotten that review multiple times.